test/entrypoint/srv_ddl.lua

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env tarantool
```

Comment:
Raw idea: run crud test suite twice.

- At the first time as currently
- The second time - use DDL with sharding key the same as primary key

Probably it's an overkill for this task

Reply:
I thought about it an decided to make a separate testsuite focused on sharding key passed using DDL module and don't parameterize the whole regression testsuite with working with/without DDL for two reasons:
- most of the CRUD source code touched by proposed patch covered by tests
- parameterize of the whole suite makes it a bit complex. To do it one need: add ability to use more than two parameters in tests(right now it is limited by two parameters only), teach existed tests to work using DDL schema...

Reply:
CRUD test suite parameterization is limited by two parameters. I propose to wait for coming parameterization in luatest - https://github.com/tarantool/luatest/pull/163 and parameterize the whole CRUD suite by ddl option in the scope of a separate ticket.

Reply:
put to backlog - https://github.com/tarantool/crud/issues/208